### PR TITLE
upgrade input group

### LINKED ISSUE #176: Reduce input-group-addon variants
only 2 variants: light-primary & white

### PR DESCRIPTION
**Pull request by JefMari**
Resolves #176 

image
- [x] Reduce input-group variants
- [x] Adjust button inside input group

### CODE REVIEW

**Comment by JefMari** [comment]
Please check @srph

**Comment by srph** [comment]
No border for the white (aka transparent) addon.

**Comment by JefMari** [comment]
Update! 
image

**Comment by srph** [comment]
How about an example where the white addon is placed on the left?

**Comment by JefMari** [comment]
Example Added!
image

File: styles/base/_form.scss [hunk 1]
```diff
@@ -137,9 +145,16 @@ label {
...
     padding: 0 8px;
-    background: $brand-gray-lt;
-    border: 1px solid $brand-gray;
+    border: 1px solid $brand-black;
     border-radius: $border-radius-base;
+    background-color: $brand-white;
+    color: $brand-black;
+  }
+  
+  > .addon.-primary-link {
```
Comment (srph) [inline styles/base/_form.scss]: Just name this as `-primary`.

File: styles/base/_form.scss [hunk 2]
```diff
@@ -137,9 +145,16 @@ label {
...
+    border-color: $brand-sidebar-lt;
+    background-color: $brand-sidebar-lt;
+    color: $brand-white;
```
Comment (srph) [inline styles/base/_form.scss:155]: Fix order.
1. `color`
2. `background`
3. `border`

File: styles/base/_form.scss [hunk 3]
```diff
@@ -153,6 +168,22 @@ label {
   > .addon.-btn {
     padding: 0;
   }
+
+  > .addon .btn {
+    height: 32px;
+    line-height: 32px;
+  }
+
+  > .addon.-btn > .btn{
```
Comment (srph) [inline styles/base/_form.scss]: Space between `.btn` and `{`.

**Comment by JefMari** [comment]
Updated!
image

File: styles/base/_form.scss (hunk 2 above)
Comment (JefMari) [inline styles/base/_form.scss:155]: Isn't this a little bit of confusing? co'z the variant name is `-primary` but the color use is `$brand-sidebar-lt` ?

**Comment by srph** [comment]
Thanks :star:

**Pull request merged**